apps/examples/testcase/tc_common.h

```diff
@@ -139,6 +139,7 @@ extern int total_fail;
 {\
 	printf("\n[%s] PASS \n", __func__); \
 	total_pass++; \
+	return; \
```

Comment:
please exclude changes which are not related to this commit

Reply:
this is the fix for one of the fs tc blocking issue

Reply:
I know that issue is due to double freeing of pointer.
Fixes are,
1. Assign NULL to pointer after first free.
2. Prevent fall through due to TC_SUCCESS_RESULT macro which intend leads to double freeing of pointer.
3. Fix tc application is blocking due to double free, which is due to some semaphore issues ,  I guess.

Reason for selecting Fix 2 is ,  while writing this tc it is assumed that TC_SUCCESS_RESULT will return, which actually right. Addition to this fix we can also add Fix 1 but in this case it doesn't have logical effect. And Fix 3 needs time so that we can check it later